### PR TITLE
Include tip box in UG in list command for all modes 

### PR DESCRIPTION
**Pull request by shadowezz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -151,6 +145,18 @@ Examples:
...
+#### Listing all contacts: `list`
+
+Shows a list of all contacts in your contact list.
+
+Format: `list`
+
+<div markdown="span" class="alert alert-primary">:bulb:
+
+**Tip:** Use this command when you want to list all your contacts back after using the `find` command. 
+Refer to [Finding contacts](#finding-contacts-find) above for details of the `find` command.
```
Comment (gabztcr) [inline docs/UserGuide.md:157]: Would hyperlinking just once in the first `find` code in line 156 and deleting the entire line 157 be better for succinctness? Here and elsewhere.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -258,6 +258,18 @@ Examples:
 * `find Homepage urgent` returns a deliverable with title `Complete homepage.` and another with description `Urgent, client is unhappy`.
 * `find Mockup` returns a deliverable with name `Complete mockup` and another with description `Add more details to existing Mockup`.
 
+#### Listing all deliverables: `list`
+
+Shows a list of all deliverables in the deliverable list.
```
Comment (gabztcr) [inline docs/UserGuide.md:263]: Can we use more "you" language, i.e. "in your deliverable list"?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -361,7 +361,19 @@ Format: `find KEYWORDS`
 
 Examples:
 * `find discuss user guide John` returns a meeting with title `discuss user guide` and another with description `with Mr John`.
-* `find Mcd0nalds` returns a meeting with title `Meet Mcd0nalds` and another with description `Cater Mcd0nalds breakfast`.
+* `find Mcdonalds` returns a meeting with title `Meet Mcdonalds` and another with description `Cater Mcdonalds breakfast`.
+
+#### Listing all meetings: `list`
+
+Lists all meetings from your meeting list.
```
Comment (gabztcr) [inline docs/UserGuide.md:368]: Can we stick to "Shows a list of all meetings in your meeting list" for consistency? (or is this too repetitive?)

File: docs/UserGuide.md (hunk 1 above)
Comment (shadowezz) [reply docs/UserGuide.md:157]: Didn't amend this cos I feel the hyperlink and `find` formatting doesn't really look well together.

**Comment by gabztcr** [comment]
We shall look into the succinctness of all our documentation again in the future. LGTM!

**Review by gabztcr — APPROVED** [review]

**Pull request merged**